code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -111,7 +111,7 @@
 		new /obj/item/clothing/accessory/holster/knife/boot/preloaded/tactical(src)
 		new /obj/item/clothing/suit/armor/vest(src)
 		new /obj/item/clothing/under/rank/head_of_security/jensen(src)
-		new /obj/item/clothing/suit/armor/hos/jensen(src)
+		new /obj/item/clothing/suit/armor/hos/sundowner(src)
```

Comment:
I considered that, but thought having an additional HoS armour to hand out every round might upset the balance.